src/core/app/index.js

```diff
@@ -5,24 +5,39 @@ import Collections from "../main/collections.styled";
 import Layout from "../../layouts/layout";
 import RenderCarousel from "../../components/carousel";
 
-import { Route, Redirect } from "react-router-dom";
-import { COLLECTIONS, COLLECTION_ID, SEARCH_RESULTS } from "../../config";
+import { Route, Redirect, Switch } from "react-router-dom";
+import {
+  COLLECTIONS,
+  COLLECTION_ID,
+  SECTION_MOVIES,
+  SEARCH_RESULTS,
+  SECTION_MOVIES_ID,
+} from "../../config";
 import VideoDetail from "../main/video-detail";
 import SearchResults from "../search/search-results.styled";
+import MoviesSection from "../movies/movies-section.styled";
```

Comment:
You should import the styled component by default.

```suggestion
import MoviesSection from "../movies/movies-section";
```